homework/03LessonThree.js

```diff
@@ -110,10 +115,14 @@ var threeByThreeArray = [
 
 // Problem 11:
 // Replace the center element of threeByThreeArray, which is 5, with something else.
+threeByThreeArray[1][1] = 'Charmander'
 
 // Problem 12:
 // create a 2 by 2 (2 rows, 2 columns) nested array
-var twoByTwoArray;
+var twoByTwoArray = [
+  [1,1],
+  [2,2]
+];
 
```

Comment:
Outside the syntax issues that prevent your mocha tests from running, all your logic is 100%. Excellent work.

![wizard](http://i.memecaptain.com/gend_images/7mS3yg.png)